<launch>
    <arg name="reward_raster_path" default=""/>

    <node name="reward_raster" type="raster_reward.py" pkg="raster_rewards" output="screen">
        <param name="reward_raster_path" value="$(arg reward_raster_path)"/>
    </node>

</launch>
